scenarios/tracking/01_training_introduction.ipynb

```diff
@@ -13,19 +13,36 @@
    "cell_type": "markdown",
```

Comment:
<p>How about:</p><p>using <s>torchvision</s> FairMOT</p><p><s>In particular, we will use </s>FairMOT <s>which</s> is a state-of-the-art tracking model with high accuracy and inference speed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>